vs-extension/troubleshooting.hbs.md

```diff
@@ -46,7 +45,7 @@ OUTPUT_TO_NULL_COMMAND = os.getenv("OUTPUT_TO_NULL_COMMAND", default=' > /dev/nu
 to
 
 ```text
-OUTPUT_TO_NULL_COMMAND_WINDOWS = os.getenv("OUTPUT_TO_NULL_COMMAND_WINDOWS", default=' > NUL ')
+OUTPUT_TO_NULL_COMMAND = os.getenv("OUTPUT_TO_NULL_COMMAND", default=' > NUL ')
```

Comment:
Nice catch, I think this could explain why Vasanth ran into [this error](https://jira.eng.vmware.com/browse/TANZUSC-2705) when trying to start Live Update: 
```
undefined: OUTPUT_TO_NULL_COMMAND (did you mean OUTPUT_TO_NULL_COMMAND_WINDOWS?)
``` 
Changed version looks good to me!